syftlib/src/main/java/org/openmined/syft/utilities/IO.kt

```diff
@@ -1,40 +1,14 @@
 package org.openmined.syft.utilities
 
 import android.util.Log
-import io.reactivex.Single
 import java.io.File
 import java.io.InputStream
 
 const val MB = 1024 * 1024
 private const val TAG = "utilities.File"
 
-class FileWriter(private val parentPath: File, fileName: String) : File(parentPath, fileName) {
-
+class FileWriter(parentPath: File, fileName: String) : File(parentPath, fileName) {
```

Comment:
We don't need this class anymore then

Reply:
Yup... It's still used in some other places so they need to be refactored too. 
I'll do it in the next iteration (little refactors every time ;) )

Reply:
The reason I created this file writer was to enforce everywhere that the parent folder is not null. You would need to do that broiler plate checking everywhere manually.

Reply:
We can leave it here by the time being. Right now it's used as a replacement for File with creation of the parent directories.

Reply:
It was used only in two places (one of them a test). I replaced it with straight file/mkdirs